### PR TITLE
release v1.12.4

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
fix time submitted display on translation progress page
fix ACF integration

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -58,6 +58,10 @@ Additional information on the Smartling Connector for WordPress can be found [he
 3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.
 
 == Changelog ==
+= 1.12.4 =
+* Fixed time submitted display on translation progress page
```
Comment (PavelLoparev) [inline readme.txt:62]: Didn't get what exactly change fixes this. This one:
```
static::$_siteCache[$site->site_id][] = (int)$site->blog_id;
```
?

File: readme.txt [hunk 2]
```diff
@@ -58,6 +58,10 @@ Additional information on the Smartling Connector for WordPress can be found [he
...
+* Fixed ACF integration: images and other non-translated media should copy to translated content
```
Comment (PavelLoparev) [inline readme.txt:63]: Didn't get what exactly change fixes this. This one:
```
static::$_siteCache[$site->site_id][] = (int)$site->blog_id;
```
?

File: readme.txt [hunk 3]
```diff
@@ -58,6 +58,10 @@ Additional information on the Smartling Connector for WordPress can be found [he
 3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.
 
 == Changelog ==
+= 1.12.4 =
```
Comment (PavelLoparev) [inline readme.txt:61]: Did we mess up with versions? I see there were `1.12.2` and `1.12.3` at the same time in the code base.

File: readme.txt (hunk 2 above)
Comment (vsolovei-smartling) [reply readme.txt:63]: Yes, it is the fix. There's a place where our ACF support tries to search for other blog ids, but due to a strict comparison, fails.

File: readme.txt (hunk 3 above)
Comment (vsolovei-smartling) [reply readme.txt:61]: I didn't update the stable tag in the readme on last release, but there is no reason not to do it now

**Pull request merged**